### PR TITLE
DR-1050 Added spf text into locals

### PR DESCRIPTION
**Pull request by mvillaseco**
Change:
 - Add "spf" text before "fromdoppler.com" text

### CODE REVIEW

**Comment by andresmoschini** [comment]
Great! I am fixing it in PROD, please merge it into master branch in place of develop.

File: src/wwwroot/locales/en-translation.js [hunk 1]
```diff
@@ -227,7 +227,7 @@
   "dkim_configuration_help_selector_title": "Your DKIM Selector:",
   "dkim_configuration_help_public_k_title": "Your DKIM Public Key:",
   "dkim_configuration_help_spf_title": "SPF",
-  "dkim_configuration_help_spf_text": "Add the following data to the ¨SPF¨ field: ¨include: fromdoppler.com¨",
+  "dkim_configuration_help_spf_text": "Add the following data to the ¨SPF¨ field: ¨include: spf.fromdoppler.com¨",
```
Comment (andresmoschini) [inline src/wwwroot/locales/en-translation.js]: Oh, there should not be a space between `include:` and `spf.spf.fromdoppler.com`

Comment (andresmoschini) [inline src/wwwroot/locales/en-translation.js]: Asking @dnoya ...

Comment (mvillaseco) [inline src/wwwroot/locales/en-translation.js]: Ok, let me know, and i'll made the changes.

**Pull request merged**